demo/src/app/core.component.ts

```diff
@@ -14,6 +14,9 @@ import { Component } from '@angular/core';
                         <li>
                         <a [routerLink]="'/case'" [routerLinkActive]="'item-bold'">View Case</a>
                         </li>
+                        <li>
+                        <a [routerLink]="'/case/filters'" [routerLinkActive]="'item-bold'">Case Create Filters</a>
```

Comment:
This is part of a future task in this epic: https://tools.hmcts.net/jira/browse/RDM-3918

Reply:
Done